_data/finance.yml

```diff
@@ -61,6 +61,14 @@ websites:
       img: quickbooks.png
       tfa: No
 
+    - name: QuickFile
+      url: http://www.quickfile.co.uk
+      twitter: quickfile
+      img: quickbooks.png
```

Comment:
My apologies, indeed I have. Damn you, copy & paste! :-)

On 18/04/15 12:58, Carlgo11 wrote:
>
> In _data/finance.yml
> <https://github.com/jdavis/twofactorauth/pull/1071#discussion_r28643525>:
>
> > @@ -61,6 +61,14 @@ websites:
> >        img: quickbooks.png
> >        tfa: No
> >  
> > +    - name: QuickFile
> > +      url: http://www.quickfile.co.uk
> > +      twitter: quickfile
> > +      img: quickbooks.png
>
> quickbooks? I think you entered the wrong file name
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdavis/twofactorauth/pull/1071/files#r28643525>.
>